Palo Alto Networks/paloalto-cortex/_meta/fields.yml

```diff
@@ -0,0 +1,28 @@
+cortexxdr.shost:
+  description: "Host of the event"
+  name: cortexxdr.shost
+  type: keyword
+cortexxdr.suser:
+  description: "User that created the event"
+  name: cortexxdr.suser
+  type: keyword
+cortexxdr.action:
+  description: "Action taken by cortexxdr"
+  name: cortexxdr.action
+  type: keyword
+cortexxdr.category:
+  description: "Category provided by cortexxdr"
+  name: cortexxdr.category
+  type: keyword
+cortexxdr.event_subtype:
+  description: "Subtype of event (provided by cortexxdr)"
+  name: cortexxdr.event_subtype
+  type: keyword
+cortexxdr.severity_human:
+  description: "Severity of the event in human readable format. Can be UNKNOWN, LOW, MEDIUM or HIGH"
+  name: cortexxdr.severity_human
+  type: keyword
+cortexxdr.original_message:
+  description: "cortexxdr message parsed into json."
+  name: cortexxdr.original_message
+  type: keyword
```

Comment:
The field ECS [`message`](https://www.elastic.co/guide/en/ecs/8.4/ecs-base.html#field-message) already holds the original message.
```suggestion
```